tests/virtualization/xen/xl_create.pm

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Export XML from virsh and create new guests in xl stack
+# Maintainer: Pavel Dostál <pdostal@suse.cz>
+
+use base "x11test";
+use xen;
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+    select_console 'x11';
+
+    x11_start_program('xterm');
+    send_key 'super-up';
+
+    foreach my $guest (keys %xen::guests) {
+        record_info "$guest", "Starting to clone $guest to $guest-xl";
+
+        # Export the XML from virsh and convert it into Xen config file
+        assert_script_run "ssh root\@javor.qam.suse.cz 'virsh dumpxml $guest > $guest.xml'";
```

Comment:
now it is javor, but later it will maybe run on another host
Variable SUT_IP could be used ?